tsl/src/partialize_finalize.c

```diff
@@ -147,6 +148,86 @@ collation_oid_from_name(char *schema_name, char *collation_name)
 	namel = lappend(namel, makeString(collation_name));
 	return get_collation_oid(namel, false);
 }
+
+#if PG14_GE
+/*
+ * Record serialized partials errata here:
+ * ============================================================================
+ * PostgreSQL versions < 14
+ * F_NUMERIC_AVG_DESERIALIZE serialized partial length = X
+ * F_NUMERIC_DESERIALIZE serialized partial length = X + 10
+ *
+ * PostgreSQL versions >= 14
+ * F_NUMERIC_AVG_DESERIALIZE serialized partial length = X + 16
+ * F_NUMERIC_DESERIALIZE serialized partial length = X + 10 + 16
+ *
+ * For F_NUMERIC_AVG_DESERIALIZE and F_NUMERIC_DESERIALIZE partials it suffices to zero out the
+ * extra bytes of pInfcount and nInfcount in the numeric aggregate combine state. Those fields
+ * are only there to support infinity and -infinity in the numeric type, which wasn't supported
+ * in older PostgreSQL versions.
+ * For more information see: https://www.postgresql.org/message-id/606717.1591924582@sss.pgh.pa.us
+ *
+ * Non-numeric aggregate functions affected:
+ * 1. var_pop(int8)
+ * 2. var_samp(int8)
+ * 3. variance(int8)
+ * 4. stddev_pop(int8)
+ * 5. stddev_samp(int8)
+ * 6. stddev(int8)
+ * ============================================================================
+ * F_NUMERIC_POLY_DESERIALIZE depends on compiler support for HAVE_INT128. As a result, recompiling
+ * the same versions of PostgreSQL and TimescaleDB with a different compiler can lead to corruption
+ * if the same database is reused.
```

Comment:
Do we do anything about this one? Looks like not, probably better to clarify that all the subsequent functions are not related to it.